optuna/distributions.py

```diff
@@ -233,6 +316,87 @@ def _contains(self, param_value_in_internal_repr: float) -> bool:
         return self.low <= value <= self.high and abs(k - round(k)) < 1.0e-8
 
 
+class IntDistribution(BaseDistribution):
+    """A distribution on integers.
+
+    This object is instantiated by :func:`~optuna.trial.Trial.suggest_int`, and passed to
+    :mod:`~optuna.samplers` in general.
+
+    Attributes:
+        low:
+            Lower endpoint of the range of the distribution. ``low`` is included in the range.
+        high:
+            Upper endpoint of the range of the distribution. ``high`` is included from the range.
+        log:
+            If ``log`` is True, this distribution is in log-scaled domain.
```

Comment:
```suggestion
            If ``log`` is :obj:`True`, this distribution is in log-scaled domain.
```

Reply:
Thank you for pointing out. I will think about a way to avoid such thing going forward.